### ISSUE TITLE
bug: tab silently fails to load if content is not wrapped in IonPage

### ISSUE THREAD

**Issue by bitjson** [ionic-team/ionic#19832]
# Bug Report

**Ionic version:**
@ionic/react: 4.11.0

**Current behavior:**
If the developer forgets to wrap the contents of a tab in `<IonPage>`, switching to the tab fails silently.

**Expected behavior:**
It would be great if a useful error message could be thrown. Especially for new users getting started, it's easy to miss how the components depend on others.

**Steps to reproduce:**
On any of the sample apps, replace `<IonPage>` in one of the tabs with `<div>`. When the user clicks the IonTabButton to switch to that tab, it silently fails to switch. (However, if the user navigates directly to the tab, it will render properly on page load.)

**Ionic info:**

```
   Ionic CLI: 5.4.5
   Ionic Framework: @ionic/react 4.11.3
   NodeJS : v10.16.3
```

**Comment by elylucas** [comment]
Thanks for the suggestion, @bitjson. In the next release, there will be a console warning if no IonPage is found to render while in development.  Thanks!

**Comment by elylucas** [comment]
fix released in 4.11.4
